### ISSUE TITLE
[request] Notifications

### ISSUE THREAD

**Issue by TristisOris** [eduardolat/pgbackweb#56]
Add options to notify about overal execution status or fails only.

**Comment by Stitch10925** [comment]
Euhm, already exists under Webhooks:

![image](https://github.com/user-attachments/assets/f72feada-7290-4e4b-84f9-b75ed90c84bf)


Maybe have a look around inside the application before posting feature requests?

**Comment by TristisOris** [comment]
webhooks is not a notifications. i mean basic smtp settings.

**Comment by SmartHome2021** [comment]
> webhooks is not a notifications...
I disagree.Yes, my notification does not come to the mail, but to telegram + zabbix, but the webhook in the role of notifications works fine.
Perhaps this option will suit you too

**Comment by SmartHome2021** [comment]
> webhooks is not a notifications. i mean basic smtp settings.

I disagree.Yes, my notification does not come to the mail, but to telegram + zabbix, but the webhook in the role of notifications works fine.
Perhaps this option will suit you too
